javaScript/JQTraversing.html

```diff
@@ -0,0 +1,40 @@
+<html>
+    <head>
+            <style>
+                    .ancestors * { 
+                      display: block;
+                      border: 2px solid lightgrey;
+                      color: lightgrey;
+                      padding: 5px;
+                      margin: 15px;
+                    }
+                    </style>
+                    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+                    <script>
```

Comment:
@Nigam-chauhan  Please add this as a separate js file and include it as an external js file.